ods_ci/tests/Tests/400__ods_dashboard/415__ods_dashboard_projects/415__ods_dashboard_projects_notebook_images.robot

```diff
@@ -0,0 +1,63 @@
+*** Settings ***
+Documentation      Suite to test the spawn of different notebook images
+Library            SeleniumLibrary
+Library            OpenShiftLibrary
+Resource           ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Projects.resource
+Resource           ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Workbenches.resource
+Resource           ../../../Resources/RHOSi.resource
+Suite Setup        Project Suite Setup
+Suite Teardown     Project Suite Teardown
+
+
+*** Variables ***
+${PRJ_TITLE}    ODS-CI DS Project Notebook Images
+${PRJ_RESOURCE_NAME}    notebook-images-ds-project
+${PRJ_DESCRIPTION}    ${PRJ_TITLE} is a test project for validating notebook images and shared by multiple tests
+${INTEL_AIKIT_APPNAME}          aikit
+${INTEL_AIKIT_OPERATOR_NAME}    Intel® oneAPI AI Analytics Toolkit Operator
+${IMG_NAME}   oneAPI AI Analytics Toolkit
+${WORKBENCH_TITLE}    aikitwb
+${PV_DESCRIPTION}    PV for AiKit workbench
```

Comment:
if this suite is supposed to contain workbenches for different images, maybe it would be better to move these image-specific variable in the tests

Reply:
changed the scope of variables to Test Case